doc/source/install.rst

```diff
@@ -24,11 +24,11 @@ The Python core team plans to stop supporting Python 2.7 on January 1st, 2020.
 In line with `NumPy's plans`_, all pandas releases through December 31, 2018
 will support Python 2.
 
-The final release before **December 31, 2018** will be the last release to
+The 0.24.x feature release will be the last release to
 support Python 2. The released package will continue to be available on
 PyPI and through conda.
 
-Starting **January 1, 2019**, all releases will be Python 3 only.
+Starting **January 1, 2019**, all feature releases will be Python 3 only.
```

Comment:
```suggestion
 Starting **January 1, 2019**, all new feature releases (> 0.24) will be Python 3 only.
```

Not sure about the exact wording, but how it is now is not strictly correct (But since the sentence above says the right thing now, I don't care too much :) )